pkg/controller/ground/bootstrap.go

```diff
@@ -310,3 +314,45 @@ func SeedAutoRenewalNodeCertificates(client clientset.Interface) error {
 		},
 	})
 }
+
+func SeedDashboardOpenstackClusterRoleBindings(client clientset.Interface) error {
+
+	err := bootstrap.CreateOrUpdateClusterRoleBinding(client, &rbac.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "kubernikus:dashboard-openstack-kubernetes-admin",
```

Comment:
This role is not really dashboard specific, no? This role will also apply for users Kubectl that authenticate against dex afais. So maybe this should be called differently? Like `openstack-kubernetes-admin`. Also the function maybe should be called. `SeedOpenStackClusterRoleBindings` os something.

Reply:
👍 Fixed